src/accelerate/utils/dataclasses.py

```diff
@@ -986,11 +987,36 @@ def set_auto_wrap_policy(self, model):
                     else:
                         transformer_cls_to_wrap.add(transformer_cls)
 
-                self.auto_wrap_policy = functools.partial(
+                auto_wrap_policy = functools.partial(
                     transformer_auto_wrap_policy,
                     # Transformer layer class to wrap
                     transformer_layer_cls=transformer_cls_to_wrap,
                 )
+
+                # In an FSDP setting PEFT models require individually wrapping unfrozen parameters
+
+                if isinstance(model, PeftModel):
+                    print("PEFT wrapping")
```

Comment:
Please remove -- or use logging with a more meaningful message.

Reply:
Will remove it.